src/LoadBalanced/Templates/Rest/Client/MethodTemplate.cshtml

```diff
@@ -18,7 +18,7 @@
 @if (!string.IsNullOrWhiteSpace(Model.Description) || !string.IsNullOrEmpty(Model.Summary))
 {
 @:/// <summary>
-@:@WrapComment("/// ", String.IsNullOrEmpty(Model.Summary) ? Model.Description.EscapeXmlComment().Replace("\n", "") : Model.Summary.EscapeXmlComment().Replace("\n", ""))
+@:@WrapComment("/// ", String.IsNullOrEmpty(Model.Summary) ? Model.Description.EscapeXmlComment().Replace(System.Environment.NewLine, "") : Model.Summary.EscapeXmlComment().Replace("\n", ""))
```

Comment:
This should be for both cases.

Reply:
don't need this anymore, will format the file to unix line ending before generate